docs/src/main/sphinx/security/cert-auth.rst

```diff
@@ -0,0 +1,108 @@
+==========================
+Certificate authentication
+==========================
+
+Trino can be configured to support certified clients seeking validation from a
+Trino server. This authentication method is only provided to support sites that
+have an absolute requirement for client authentication *and already have* client
+certificates for each client. Sites in this category have an existing PKI
+infrastructure, possibly including an onsite CA.
+
+This feature is *not* appropriate for sites that would need to generate a set of
+client certificates in order to use this authentication method. Starburst
+recommends instead configuring one of the password-based authentication methods,
+including :doc:`password-file`, :doc:`ldap`, and :doc:`salesforce`; or choosing
+an :doc:`OAuth2 <oauth2>`-compatible authentication vendor.
+
+
+Using certificate authentication
+--------------------------------
+
+With certificate authentication enabled, the validation sequence is the
+following:
+
+* A client verfies the coordinator's certificate, and makes an attempt to
```

Comment:
Isnt there a step missing before that? Like roughly

* User initiates authentication by connecting to Trino using TLS/HTTPS with a client tool such as the CLI.
* Trino coordinator provides the the certificate to the client.
* Client verifies the certificate and in the secure connection supplies its certificate.

Reply:
Those are server auth steps, and this page is about client auth. Should we really muddy the waters by telling the full and complete story, or just stick to the client auth steps?

Reply:
Section reworked to include both client to server (standard for all auth types) and server to client auth.